v4/source/introduction.yaml

```diff
@@ -214,23 +214,205 @@ tags:
 
       The list of Mattermost WebSocket events are:
       - added_to_team
+        ```
+        {
+          event: 'added_to_team',
```

Comment:
Can you be consistent around when we use quotes on the keys? Here `event` doesn't have quotes but the next event it does